spot_rl_experiments/spot_rl/skills/atomic_skills.py

```diff
@@ -470,10 +498,14 @@ def __init__(self, spot, config=None, use_mobile_pick=False) -> None:
         self.env = SpotGazeEnv(self.config, spot, use_mobile_pick)
 
     def set_pose_estimation_flags(
-        self, enable_pose_estimation: bool = False, enable_pose_correction: bool = False
+        self,
+        enable_pose_estimation: bool = False,
+        enable_pose_correction: bool = False,
+        mesh_name: str = "",
```

Comment:
what is this? do we need this in this PR?

Reply:
I don't think Vyshnav added this. Let's remove it from this PR

Reply:
self.mesh_name was used for pose estimation task (one where we used 3D meshes obtained from Kiri engine to perform pose estimation). I can remove it if not required ( I didn't add this either). Will remove all instance of self.mesh as well in this case.